docs/BUILD

```diff
@@ -9,20 +9,24 @@ exports_files(["protodoc_manifest.yaml"])
 
 envoy_package()
 
-# TODO(phlax): fix failing/excluded configs
-# the following config only fails on windows:
-#       dns-cache-circuit-breaker: "Error: unable to read file: /etc/ssl/certs/ca-certificates.crt"
-
 filegroup(
     name = "configs",
     srcs = glob(
-        ["root/**/*.yaml"],
+        [
+            "root/**/*.yaml",
+            "root/**/*.pb",
+        ],
         exclude = [
-            "root/intro/_include/life-of-a-request.yaml",
+            # TODO(phlax/windows-dev): figure out how to get this working on windows
+            #      "Error: unable to read file: /etc/ssl/certs/ca-certificates.crt"
```

Comment:
I'll open an issue for this, it could dovetail with the idea of Envoy tapping into the trusted system cert pool as well potentially?

Reply:
@phlax feel free to chime in with any more details or ideas: https://github.com/envoyproxy/envoy/issues/13596

Reply:
cool, will do